spa/src/pages/Repositories/index.tsx

```diff
@@ -0,0 +1,78 @@
+import Button from "@atlaskit/button";
+import { Wrapper } from "../../common/Wrapper";
+import ArrowLeftIcon from "@atlaskit/icon/glyph/arrow-left";
+import { useNavigate } from "react-router-dom";
+import PageHeader from "@atlaskit/page-header";
+import { token } from "@atlaskit/tokens";
+import { css } from "@emotion/react";
+import Form, { Field } from "@atlaskit/form";
+import Textfield from "@atlaskit/textfield";
+import SearchIcon from "@atlaskit/icon/glyph/search";
+import Select, { ValueType } from "@atlaskit/select";
+import { Inline } from "@atlaskit/primitives";
+
+const Repositories = () => {
+	const navigate = useNavigate();
+	const titleStyle = css`
```

Comment:
/** @jsxImportSource @emotion/react */
currently its necessary to add above annotation to the page at the top, 
If you are using 
import { css } from "@emotion/react";

Reply:
Removed the page in its entirety